File: BMI Calculater.js
function calculateBMI() {
  // Get the user's height and weight.
  height = parseFloat(prompt("Enter your height in meters: "));
  weight = parseFloat(prompt("Enter your weight in kilograms: "));

  // Calculate the BMI.
  bmi = weight / (height ** 2);
  console.log("Your BMI is: " + bmi);
  // Return the BMI.
  return bmi;

    
}

// Function to interpret values of BMI.
function interpretBMI(bmi) {
  if (bmi < 18.5) {
    console.log("Underweight");
  } else if (18.5 <= bmi < 25) {
    console.log("Normal weight");
  } else if (25 <= bmi < 30) {
    console.log("Overweight");
  } else {
    console.log("Obese");
  }
}



   bmi = calculateBMI();
   interpretBMI(bmi);

File: BMI Calculater.py
#BMI Program
#function to receive height and weight from user ,Calculate and return  BMI

def calculate_bmi ():
    height = float(input("Enter your height in m: "))
    weight = float(input("Enter your weight in km: "))

    bmi = weight / (height ** 2)
    print(f"Your BMI is : {bmi}")
    return bmi

# Function to interpret values of BMI
def interpret_bmi(bmi):
    if bmi <18.5:
        print("Underweight")
    elif 18.5  <= bmi < 25:
        print("Normal weight")
    elif 25 <= bmi < 30 :
        print("Overweight")
    else :
        print("Obese")


if __name__ == "__main__":
    bmi = calculate_bmi()
    interpret_bmi(bmi)
